devtools/src/panel/components/PiPContainer.vue

```diff
@@ -1,8 +1,10 @@
 <script setup lang="ts">
-import { useColorMode } from '@vueuse/core'
+import { useColorMode, useLocalStorage } from '@vueuse/core'
 import { computed } from 'vue'
+import ResizablePanel from './ResizablePanel.vue'
 
 const { isPip } = defineProps<{
+  id: string
```

Comment:
The id is needed for CSS.
Instead, adding `v-bind="$attrs"` to the aside is the way. This keeps this inheritance of attributes with a multi root node component